docs/workshop/03_ide.md

```diff
@@ -221,13 +221,15 @@ In `IBM I Project Explorer`, right-click on `arcad-example` and select `Configur
 
 ---
 
-
+### Initialize project for ARCAD
 
 <!-- panels:start -->
 
 <!-- div:left-panel -->
 
-Enter an application code that does NOT already exist, e.g. `SAMCO`.  Press `Enter`
+Enter an application code that does NOT already exist, e.g. `SAMCO`.  Press `Enter`.
+
+**Note:** The project is initialized only once.  If already initialized, no further prompts appear and  `arcad-example` will now have an `ARCAD` node in `IBM I Project Explorer`.
```

Comment:
The `I` should be lowercase in `IBM i`

Reply:
it is the same as in the UI (which automatically uppercases the first letter of each word)